tidb-cloud/premium/connect-to-premium-via-public-connection.md

```
@@ -36,10 +36,20 @@ To connect to a {{{ .premium }}} instance via public connection, take the follow
 
     If you have not configured the IP access list, click **Configure IP Access List** or follow the steps in [Configure an IP Access List](/tidb-cloud/premium/configure-ip-access-list-premium.md) to configure it before your first connection.
 
+<CustomContent plan="premium">
+
 4. Click **CA cert** to download CA cert for TLS connection to {{{ .premium }}} instances. The CA cert supports TLS 1.2 by default.
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/2/3

(MD029, ol-prefix)